src/porepy/geometry/intersections.py

```diff
@@ -1440,31 +1458,50 @@ def segments_polygon(
 
 def segments_polyhedron(
     start: np.ndarray, end: np.ndarray, poly: np.ndarray, tol: float = 1e-5
-) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+) -> tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
     """Compute the intersection from line segments to the interior of a convex polyhedron.
```

Comment:
Some of the lines in this docstring are longer than 88 chars. In line 1972 there is another comment longer than 88 chars.